contrib/systemd/etcd2-backup-coreos/README.md

```diff
@@ -234,18 +234,18 @@ for node in q{2..3};do
 done
 ```
 
-Once you've verifed the cluster has migrated properly, start and enable `etcd2-backup.timer` on at least one node.
+After confirming the cluster has migrated properly, start and enable `etcd2-backup.timer` on at least one node.
 
 ```sh
 ssh core@q1 "sudo systemctl enable etcd2-backup.service && sudo systemctl start etcd2-backup.service"
 ```
 
-You should now have periodic backups going to: `s3://etcd2-backup-bucket/backups/<q1-machine-id>`
+There should now be periodic backups going to: `s3://etcd2-backup-bucket/backups/<q1-machine-id>`
 
 ## Words of caution
 
-1. Notice the `sleep 10` commands that follow starting `etcd2-join.service` and then `etcd2.service`. This sleep is there to allow the member that joined to cluster time to catch up on the cluster state before we attempt to add the next member. This involves sending the entire snapshot over the network. If you're dataset is large, or the network between nodes is slow, or your disks are already bogged down, etc- you may need to turn the sleep time up.
+1. Notice the `sleep 10` commands that follow starting `etcd2-join.service` and then `etcd2.service`. This sleep is there to allow the member that joined to cluster time to catch up on the cluster state before we attempt to add the next member. This involves sending the entire snapshot over the network. If the dataset is large, or the network between nodes is slow, or disks are already bogged down, etc- try increasing the sleep.
```

Comment:
`If the dataset is large, or the network between nodes is slow, or disks are already bogged down, etc- try increasing the sleep.` 
=>
`If the dataset is large, the network between nodes is slow, the disks are already bogged down, or etc., try increasing the sleep.`?